src/entities/exportAccount.ts

```
@@ -0,0 +1,194 @@
+  private normalizeUrl(path: string): string {
+    const base = homeUrl.endsWith("/") ? homeUrl : `${homeUrl}/`;
+    return new URL(path, base).toString();
+  }
+
+  serializeFollowing(followingAccounts: schema.Follow[]) {
+    return {
+      "@context": "https://www.w3.org/ns/activitystreams",
+      id: "following_accounts.json",
+      type: "OrderedCollection",
+      orderedItems: followingAccounts.map((account) => ({
+        account: this.normalizeUrl(`accounts/${account.followingId}`),
```

Biome (1.9.4)

[error] This property value named account is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named account


(lint/suspicious/noDuplicateObjectKeys)